modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIFilterRelevantObjectEntryModelListenerTest.java

```diff
@@ -21,14 +21,16 @@
 import com.liferay.portal.kernel.test.util.RandomTestUtil;
 import com.liferay.portal.kernel.util.Http;
 import com.liferay.portal.test.rule.FeatureFlags;
+
 import org.junit.Assert;
 import org.junit.Test;
 
 /**
  * @author Alberto Javier Moreno Lage
  */
 @FeatureFlags({"LPS-167253", "LPS-184413", "LPS-186757"})
-public class APIFilterRelevantObjectEntryModelListenerTest extends BaseTestCase {
+public class APIFilterRelevantObjectEntryModelListenerTest
```

Comment:
I guess this change was performed by SF. Next time include it in the original commit where you modified / created that class to avoid mixing different things in the same commit.